source/reporting/index.html.md.erb

```diff
@@ -121,132 +123,32 @@ Example response:
-
-#### corporate_card_surcharge
-
-`corporate_card_surcharge` shows the surcharge amount in pence if you've [set up adding corporate card fees](/corporate_card_surcharges/#add-corporate-card-fees).
-
-#### created_date
-
-`created_date` is when you created the payment, in [ISO
-8601 format](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard).
-
-#### delayed_capture
-
-`delayed_capture` is `true` if you're [controlling how long it takes GOV.UK Pay to take ('capture') a payment](/delayed_capture/#delay-taking-a-payment).
-
-#### fee
-
-If you use GOV.UK’s payment service provider (PSP), `fee` is the [PSP transaction fee](/reporting/#psp-fees) we took from the amount your user paid.
-
-`fee` only appears after we have taken ('captured') the payment from the user or if their payment fails after they have submitted their card details.
-
-#### metadata
-
-The `metadata` object includes any metadata keys and values you added when creating a payment. You can [add custom metadata to new payments](https://docs.payments.service.gov.uk/custom_metadata/#add-custom-metadata) to help with reconciliation and reporting.
-
-#### moto
-
-`moto` is `true` if you [took the payment over the phone](/moto_payments/moto_api).
-
-#### net_amount
-
-If your PSP is Stripe, `net_amount` is the amount that Stripe will pay into your account after GOV.UK Pay take the [PSP transaction fees](https://docs.payments.service.gov.uk/reporting/#psp-fees).
-
-`net_amount` can be a negative number if [Stripe charges fees on a failed payment](/reporting/#psp-fees).
-
-#### payment_id
-
-`payment_id` is the unique ID that GOV.UK Pay automatically generated for this payment.
-
-#### payment_provider
-
-`payment_provider` is the name of your PSP, or `sandbox` if you're using a test ('sandbox') account.
-
-#### refund_summary
-
-You can use `refund_summary` to [check if you can refund a payment](/refunding_payments/#check-if-you-can-refund-a-payment).
-
-#### return_url
-
-`return_url` is the URL that your service hosts for your user to return to, after their payment journey on GOV.UK Pay ended.
-
-#### settlement_summary
-
-Use `settlement_summary` to check when your PSP:
+The response contains:
 
-- [took the payment from your user's account](#checking-when-your-psp-took-a-payment)
-- [sent a payment to your bank account](/integrate_with_govuk_pay/#checking-when-your-psp-sent-a-payment), if your PSP is Stripe
+* when you created the payment (`created_date`)
+* the amount of the payment, in pence (`amount`)
+* the status of the payment (`state`)
+* details you sent when you created the payment (`reference`, `description`, `metadata`)
 
-#### state
+You can see definitions and possible values for every attribute in this response in our [API reference for this endpoint](/api_reference/single_payment_reference). 
```

Comment:
```suggestion
You can see definitions and possible values for every attribute in this response in our [API reference for a single payment](/api_reference/single_payment_reference). 
```

Reply:
including different wording to be more accessible ... have assumed it's for a single payment API reference but please tweak if not right

Reply:
Agree, it could be clearer. Will change it a bit.